components/scream/src/interface/atm_comp_mct.F90.old

```diff
@@ -0,0 +1,119 @@
+module atm_comp_mct
+
+  ! These are used across the module, so we issue a single use clause here
```

Comment:
A good version-control system like Git should mean we don't need to keep around .old files. Is there something special that means we have to keep this?

Reply:
yes, I'll delete the file.